src/Utils/utils.ts

```diff
@@ -224,3 +224,13 @@ export const formatCurrency = (price: number) =>
     style: "currency",
     currency: "INR",
   });
+
+export const fahrenheitToCelcius = (x: any) => {
+  const t = (Number(x) - 32.0) * (5.0 / 9.0);
+  return String(t.toFixed(1));
+};
+
+export const celciusToFahrenheit = (x: any) => {
+  const t = (Number(x) * 9.0) / 5.0 + 32.0;
+  return String(t.toFixed(1));
+};
```

Comment:
Why are the types `any`? Isn't it better to strictly type it to `number`?
Converting `number` to `string` and `string` to `number` can be done from the side that invokes this function instead if required.

Reply:
This function was written previously, I just moved it to utils.ts. But yes it can be better to type x as number.